dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -346,12 +346,15 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink, public virtual
 
   // RTPS reliability support for local writers:
 
+  typedef OPENDDS_MAP(CORBA::ULong, RTPS::FragmentNumberSet) RequestedFragMap;
```

Comment:
I don't like the idea of making the key a struct... we'd have to implement a custom comparison operation, since we can't define FragmentNumber_t::operator<(). How about if we add a typedef of ULong called FragmentNumberValue and make that the key type here? Should help with readability, but won't change the code that uses it?